test/Issue14.hs

```diff
@@ -0,0 +1,11 @@
+module Issue14 where
+
+constid :: a -> b -> b
+constid x = \ x -> x
+
+sectionTest₁ :: Integer -> Integer -> Integer
+sectionTest₁ n = \ section -> section + n
+
+sectionTest₂ :: Integer -> Integer -> Integer
+sectionTest₂ section = \ section -> section + section₁
+
```

Comment:
This seems to have been committed by mistake, right? (it's also an incorrect compilation `!= test/golden/Issue14.hs`

Reply:
Oh, well spotted. Must have been left over from one of the test runs.